docs/book/getting-started/deploying-zenml/upgrades.md

```diff
@@ -0,0 +1,4 @@
+---
+description: How to upgrade ZenML
```

Comment:
What is this?

Reply:
Would you believe this was dragged in by the proverbial cat ? :)